src/scripts/ui/system-status.js

```diff
@@ -0,0 +1,100 @@
+window.addEventListener('DOMContentLoaded', () => {
+  const $headerStatus = document.querySelector('.header-system-status-container');
+  const $headerStatusMessage = document.querySelector('.header__status-message');
+
+  const $operationalIcons = document.querySelectorAll('.operational');
+  const $degradedPerformanceIcons = document.querySelectorAll('.degraded-performance');
+  const $partialServiceDisruptionIcons = document.querySelectorAll('.partial-service-disruption');
+  const $serviceDisruptionIcons = document.querySelectorAll('.service-disruption');
+  const $securityEventIcons = document.querySelectorAll('.security-event');
+  const $showStatusIcon = document.querySelector('.show-status-icon');
+
+  // https://kb.status.io/developers/status-codes/
+  const status = {
+    DEGRADED_PERFORMANCE: {
+      icons: $degradedPerformanceIcons,
+      message: 'Degraded Performance - We are currently experiencing degraded performance in some of our web services.',
+    },
+    OPERATIONAL: {
+      icons: $operationalIcons,
+      message: 'testing',
+    },
+    PARTIAL_SERVICE_DISRUPTION: {
+      icons: $partialServiceDisruptionIcons,
+      message: 'Partial Service Disruption - Some of our web services are temporarily unavailable.',
+    },
+    SECURITY_EVENT: {
+      icons: $securityEventIcons,
+      message: 'Security Event - We are currently mitigating issues relating to some of our web services.',
+    },
+    SERVICE_DISRUPTION: {
+      icons: $serviceDisruptionIcons,
+      message: 'Service Disruption - Our web services are temporarily unavailable.',
+    },
+  };
+
+  const setSystemStatus = ($icons, message) => {
+    if ($showStatusIcon) {
```

Comment:
@kevcenteno I'm trying to figure out how to remove the previous icon if the status ever changes. For some reason this line always consoles `no icon found` even if there is already a `$showStatusIcon` class on the page somewhere.
![Screen Shot 2022-01-18 at 8 58 28 AM](https://user-images.githubusercontent.com/52248962/149951299-2ac212f9-528d-49fc-b587-eaaf87ff8479.png)
![Screen Shot 2022-01-18 at 8 58 34 AM](https://user-images.githubusercontent.com/52248962/149951306-e7f30d65-1901-476f-a0f1-c3ad5a543249.png)
.

Reply:
What you can do is hide all the icons immediately when the function executes so that you start a blank slate.  Effectively, the rendering process would look like something like this:

```
setSystemStatus is called
  All icons are hidden
  Determine which icon should be displayed
  Display the appropriate icon
```

Reply:
Ah, thank you. This is ready for another round of code review.

I'm not sure if we want to implement the unit testing in this PR or a follow up one, though.